frontend/src/components/utils/forms/DayPickerInputField.tsx

```diff
@@ -13,38 +13,44 @@ export type DayPickerInputFieldProps = {
     parseFormats?: string | string[];
     displayFormat?: string;
     todayButton?: boolean;
-    fieldProps?: string | FieldHookConfig<Date>;
-};
+} & FieldAttributes<any>;
 
 /**
- * This component builds a bridge between `DayPickerInput` and Formik. The date time is always set at the beginning
- * of the day (00:00).
+ * A `DayPickerInput` component tied to a Formik field.
  *
- * The value of the field (accessible in `values[name]`) is always a Date object, but it may an `Invalid Date` object.
+ * The value of the field (accessible in `values[name]`) is a `Date` object or
+ * `undefined` if the `Date` is invalid (i.e. when the input contains something
+ * which could not be parsed).
  *
- * You may also be interested in `DayPickerInputFormGroup` which puts this component in a `FormGroup`.
+ * Please note that due to an unexplained behaviour of Formik when clicking on
+ * Submit several times in a row, the condition for showing the feedback is that
+ * `errors[name]` is set and `feedback` is true (`touched` has thus no effect).
  *
  * @param name the name of the control, used to access its value in Formik.
- * @param feedback defaults to true. If `true`, the `is-invalid` class is given to the input component when needed.
- * @param parseFormats defaults to `["DD/MM/YYYY"]`. The formats used to parse the date.
- * @param displayFormat defaults to `"DD/MM/YYYY"`. The format used to display the date.
- * @param todayButton defaults to `false`. If `true`, a `todayButton` will be displayed in the day picker.
- * @param fieldProps passed to Formik's `useField`.
+ * @param feedback defaults to true. If `true`, the `is-invalid` class is given
+ * to the input component when needed.
+ * @param parseFormats defaults to `["DD/MM/YYYY"]`. The formats used to parse
+ * the date.
+ * @param displayFormat defaults to `"DD/MM/YYYY"`. The format used to display
+ * the date.
+ * @param todayButton defaults to `false`. If `true`, a `todayButton` will be
+ * displayed in the day picker.
+ * @param props passed to Formik's `useField`.
  */
 export const DayPickerInputField = ({
     name,
     feedback = true,
     parseFormats = ["DD/MM/YYYY"],
     displayFormat = "DD/MM/YYYY",
     todayButton = false,
-    ...fieldProps
+    ...props
 }: DayPickerInputFieldProps) => {
     const [field, meta, helper] = useField<Date | undefined>({
         validate: (value) =>
             value === undefined
-                ? "Veuillez entrer une date au format JJ/MM/YYYY"
+                ? "Veuillez entrer une date au format JJ/MM/YYYY."
                 : undefined,
```

Comment:
`value  || "Veuillez entrer une date au format JJ/MM/YYYY."`

Reply:
Not the same => we never want to return `value`, but either the text or `undefined`.